config/app.default.php

```diff
@@ -88,7 +88,7 @@
      */
     'Cache' => [
         'default' => [
-            'className' => 'File',
+            'className' => 'Cake\Cache\Engine\FileEngine',
```

Comment:
Even better would be using `FileEngine::class` instead of string. Same for other classes.